pkgs/tools/misc/pridefetch/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, lib, python39, zip }: let
```

Comment:
`lib` usually comes first. Also, I think it's better not to depend on a specific minor version of Python. `python3` should be fine.